packages/core/src/Step.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Text from './Text'
+import Icon from 'pcln-icons'
+
+const StyledFlex = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Step({ className, active, completed, children, onClick }) {
+  const color = active || completed ? 'primary' : 'text.light'
+
+  return (
+    <StyledFlex className={className} alignItems="center" onClick={onClick}>
+      {completed && (
+        <Icon name="check" color="primary" size={16} mr={1} legacy={false} />
+      )}
+      <Text color={color} bold={active} fontSize={1}>
+        {children}
+      </Text>
+    </StyledFlex>
+  )
+}
+
+Step.displayName = 'Step'
+
+Step.propTypes = {
+  className: PropTypes.string,
+  active: PropTypes.bool,
+  completed: PropTypes.bool,
+  children: PropTypes.node,
```

Comment:
i feel like children should be required. thoughts?

Reply:
Probably not a bad idea, unless we see value in supporting an icon-only stepper, but we'd want have to make more changes in that case anyway. Seems like it should be required.

Reply:
I think that's a fair requirement 👍